creating_citation_counts_tsv/scripts/summarize_gene_citations_all_species.py

```diff
@@ -0,0 +1,335 @@
+        if refGene_file != "gene_info" and "gene2pubmed":
+            # Adding gene's reference information to the list created from getting_gene_info per gene and saving it to the refGene variable
+            refGene = (sc.textFile(f"creating_citation_counts_tsv/data/{species}/{refGene_file}")
+                    .filter(lambda x: x[0] != '#')
+                    .map(lambda x: x.split('\t'))
+                    .filter(lambda x: 'transcript' in x[2])
+                    .map(lambda x: (str(x[8]).split(";")[0].replace("gene_id ", "").replace('"', '').upper(), (int(x[3]), int(abs(int(x[3])-int(x[4])))))).join(gene_info).map(lambda x: (x[0], (x[1][1][1], x[1][0][0], x[1][0][1], "#73af42", x[1][1][2], x[1][1][3]))).reduceByKey(lambda a, b: a).map(lambda x: (x[0], x[1][0], x[1][1], x[1][2], "#73af42", x[1][4], x[1][5])))
+            
+            # Outputing the refGene variable
+            print(refGene.take(1))
+            
+            # Returning the refGene variable
+            return refGene
+
+def translate_disease(disease):
+    """ 
+    Getting the common translated disease per disease given
+    """
+    
+    # Dictionary of common translated disease to disease
+    disease_translations = {
+    'Neoplasms' : 'Cancer',
+    'Musculoskeletal Diseases' : 'Muscle and Bone Diseases',
+    'Digestive System Diseases' : 'Gastrointestinal Diseases',
+    'Stomatognathic Diseases' : 'Mouth and Jaw Diseases',
+    'Respiratory Tract Diseases' : 'Respiratory System Diseases',
+    'Otorhinolaryngologic Diseases' : 'Ears, Throat and Nose Diseases',
+    'Nervous System Diseases' : 'Nervous System Diseases',
+    'Eye Diseases' : 'Eye Diseases',
+    'Male Urogenital Diseases' : 'Male Genital Organ Diseases',
+    'Female Urogenital Diseases and Pregnancy Complications' : 'Female Genital Organ and Pregnancy Diseases',
+    'Cardiovascular Diseases' : 'Heart and Blood Diseases',
+    'Hemic and Lymphatic Diseases' : 'Blood Diseases',
+    'Congenital, Hereditary, and Neonatal Diseases and Abnormalities' : 'Inherited and Newborn Diseases',
+    'Skin and Connective Tissue Diseases' : 'Skin Diseases',
+    'Nutritional and Metabolic Diseases' : 'Nutrition and Metabolism Related Diseases',
+    'Endocrine System Diseases' : 'Hormone Related Diseases',
+    'Immune System Diseases' : 'Immune System Diseases',
+    'Disorders of Environmental Origin' : 'Environmental Disorders',
+    'Occupational Diseases' : 'Work-Related Diseases',
+    'Substance-Related Disorders' : 'Substance-Related Disorders',
+    'Wounds and Injuries' : 'Wounds and Injuries',
+    'Behavior and Behavior Mechanisms' : 'Behavior',
+  }
+    try: 
+        # Check to see if the disease is disease_translations Dictionary
+        # Return common translated disease (If found)
+        return disease_translations[disease]
+    except:
+        # Return 'Other' (If not found)
+        return 'Other'
+
+def make_request(endpoint, params=None):
```

Comment:
A more glanceable and specific function name would help understandability:

```suggestion
def request_disgenet(endpoint, params=None):
```